ipam/paxos/participant.go

```diff
@@ -7,6 +7,12 @@ type Participant interface {
 	Propose()
 	Think() bool
 	Consensus() (bool, AcceptedValue)
+	Status() *Status
```

Comment:
Not thrilled about this - the current status code has been carefully separated out of the things on which information is being gathered, and all of the dependencies go from the status code to the observed structures. This introduces a dependency in the other direction